### PR TITLE
changed http urls in tests & code

### PR DESCRIPTION
**Pull request by wdwatkins**
Changed all the URLs in tests and functions.  Documentation and example URLs still need to be changed.  #314

### CODE REVIEW

File: R/01-webdata-obj.R [hunk 1]
```diff
@@ -117,30 +117,30 @@ setMethod("webdata", signature("ANY"), function(.Object, ...) {
...
+              url = 'https://cida.usgs.gov/thredds/dodsC/prism_v2',
               variables = 'ppt'),
        'iclus' = 
-         list(url = 'http://cida.usgs.gov/thredds/dodsC/iclus/hc',
+         list(url = 'https://cida.usgs.gov/thredds/dodsC/iclus/hc',
               variables = 'housing_classes_iclus_a1_2010'),
        'daymet' = 
          list(times = as.POSIXct(c('2000-01-01 00:00:00','2001-01-01 00:00:00')),
-              url = 'http://thredds.daac.ornl.gov/thredds/dodsC/daymet-agg/daymet-agg.ncml',
+              url = 'https://thredds.daac.ornl.gov/thredds/dodsC/daymet-agg/daymet-agg.ncml',
```
Comment (jread-usgs) [inline R/01-webdata-obj.R]: This isn't a usgs url, so it might not be changing

File: tests/testthat/test-dodsReplace.R [hunk 2]
```diff
@@ -2,9 +2,9 @@ context("dodsReplace")
 
 
 test_that("prism is replaced", {
-  expect_equal(geoknife:::dodsReplace('http://cida.usgs.gov/thredds/dodsC/prism_v2'),'dods://cida.usgs.gov/thredds/dodsC/prism_v2')
+  expect_equal(geoknife:::dodsReplace('https://cida.usgs.gov/thredds/dodsC/prism_v2'),'dods://cida.usgs.gov/thredds/dodsC/prism_v2')
 })
 
 test_that("nldas is replaced", {
-  expect_equal(geoknife:::dodsReplace('http://hydro1.sci.gsfc.nasa.gov/dods/NLDAS_FORA0125_M.002'),'dods://hydro1.sci.gsfc.nasa.gov/dods/NLDAS_FORA0125_M.002')
+  expect_equal(geoknife:::dodsReplace('https://hydro1.sci.gsfc.nasa.gov/dods/NLDAS_FORA0125_M.002'),'dods://hydro1.sci.gsfc.nasa.gov/dods/NLDAS_FORA0125_M.002')
```
Comment (jread-usgs) [inline tests/testthat/test-dodsReplace.R]: also is not a USGS url

Comment (wdwatkins) [inline tests/testthat/test-dodsReplace.R]: mm, thought I checked those

**Comment by jread-usgs** [comment]
@wdwatkins can you build the documentation (it is all roxygen2) and knit the vignettes as well on your PR?
Also, probably want to increment patch version number in the DESCRIPTION

**Comment by wdwatkins** [comment]
@jread-usgs should the vignette htmls be checked in to the repo?  They currently are not

**Comment by jread-usgs** [comment]
yes, they are here: https://github.com/USGS-R/geoknife/tree/master/inst/doc

the result of 
```r
devtools::build_vignettes()
```

**Comment by wdwatkins** [comment]
Ok, that worked.  They just built the html in the vignettes folder when I built with `build vignettes` checked.

**Comment by wdwatkins** [comment]
The readme will need to be reknitted as well once the https stuff is working

**Comment by jiwalker-usgs** [comment]
#317 includes these changes

**Pull request merged**